### PR TITLE
OCPBUGS-10696 - SNO clusters upgrade with TALM docs are missing infor…

### PR DESCRIPTION
**Pull request by slovern**
…mation

Version(s):
- 4.14
- 4.13
- 4.12
- 4.11

Issue: https://issues.redhat.com/browse/OCPBUGS-10696

Link to docs preview: https://64284--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades#talo-policies-concept_cnf-topology-aware-lifecycle-manager

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: snippets/cnf-example-upgrade-policy.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+:_content-type: SNIPPET
+.Example upgrade policy
+[source,yaml,subs="attributes+"]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: upgrade
+  namespace: default
```
Comment (jc-rh) [inline snippets/cnf-example-upgrade-policy.adoc:9]: Use anything but `default`

**Comment by josclark42** [comment]
LGTM

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by slovern** [comment]
Thanks for the review @jeana-redhat

**Comment by jeana-redhat** [comment]
@slovern 4.11 needs manual cherrypicking, I'll merge 4.12+ when they build :+1:

**Pull request merged**